src/ui/common/src/utils/operators.ts

```diff
@@ -74,7 +74,8 @@ export type ExtractParameters =
   | GoogleSheetsExtractParams;
 
 export type RelationalDBExtractParams = {
-  query: string;
+  query?: string;
+  queries?: string[];
```

Comment:
My argument for making this a single list is that this stuff propagates to every one of our surfaces, and would rather pay the upfront migration cost now.